spec/presenters/default_deadline_date_time_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe DefaultDeadlineDateTime do
+  before(:all) do
+    BusinessTime::Config.holidays << Date.parse('2016-09-05')
```

Comment:
let's do this in each one! I know it's more lines of code but when the setup is outside of the test it is harder to know what the dependencies are. Also, I am guessing we don't need this config for each test.